### PR TITLE
Update integrations.md

### PR DESCRIPTION
**Pull request by chriselion**
As requested on the original PR (https://github.com/dotnet/format/pull/872)

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -46,3 +46,18 @@ Add following to your build file:
...
 These instructions originally authored by [leotsarev](https://github.com/joinrpg/joinrpg-net/).
+
+
+## [pre-commit.com](https://pre-commit.com/) hook to reformat 
+
+Add the following block to the `repos` section of your `.pre-commit-config.yaml` file:
+
+```yaml
+-   repo: https://github.com/dotnet/format
+    rev: "v5.0.211103"  # Or a more recent commit sha
```
Comment (chriselion) [inline docs/integrations.md:57]: I'm a little unclear what the rev here should be (I copied this from the PR and updated to the latest public release). Do we need to do a release with the `.pre-commit-hooks.yaml` change first, then update this to be that release tag?

Comment (rkm) [reply docs/integrations.md:57]: The latest release/tag at the time of writing is probably ok. If you look at e.g. [pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks/commit/6e2418c5521b7d606e72914dced3253f9ace1205#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5), anthony updates the referenced version manually on each release. If we don't want to do that, we could recommend starting with `rev: ""` and running `autoupdate` to bring-in the current version?

Comment (chriselion) [reply docs/integrations.md:57]: Sounds good, this is basically what Anthony recommended in discord. We can update the docs ones there's a tagged release that contains the hook.

**Review by rkm — APPROVED** [review]
looksgood.gif

**Pull request merged**